nilearn/regions/region_extractor.py

```diff
@@ -0,0 +1,324 @@
+        n_voxels across all maps. The probability of chance of nonzero voxels
+        survived after taking ratio to the total number of brain voxels will be kept.
+
+    extractor: string {'connected_components', 'local_regions'} default 'local_regions', optional
+        If 'connected_components', each component/region in the image is extracted
+        automatically by labelling each region based upon the presence of unique
+        features in their respective regions.
+        If 'local_regions', each component/region is extracted based on their
+        maximum peak value to define a seed marker and then using random walker
+        segementation algorithm on these markers for region separation.
+
+    standardize: boolean, True or False, default False, optional
+        If True, the time series signals are centered and normalized by
+        putting their mean to 0 and variance to 1. Recommended to
+        set as True if signals are not already standardized.
+        passed to class NiftiMapsMasker.
+
+    detrend: boolean, True or False, default False, optional
+        This parameter is passed to nilearn.signal.clean basically
+        indicates whether to detrend timeseries signals or not.
+        passed to class NiftiMapsMasker.
+
+    low_pass: float, default None, optional
+        This value will be applied on the signals by passing to signal.clean
+        Please see the related documentation signal.clean for more details.
+        passed to class NiftiMapsMasker.
+
+    high_pass: float, default None, optional
+        This value will be applied on the signals by passing to signal.clean
+        Please see the related documentation signal.clean for more details.
+        passed to NiftiMapsMasker.
+
+    t_r: float, default None, optional
+        Repetition time in sec. This value is given to signal.clean
+        Please see the related documentation for details.
+        passed to NiftiMapsMasker.
+
+    memory: instance of joblib.Memory, string, default None, optional
+        Used to cache the masking process. If a string is given, the path
+        is set with this string as a folder name in the directory.
+        passed to NiftiMapsMasker.
+
+    memory_level: int, default 0, optional
+        Aggressiveness of memory catching. The higher the number, the higher
+        the number of functions that will be cached. Zero mean no caching.
+        passed to NiftiMapsMasker.
+
+    verbose: int, default 0, optional
+        Indicates the level of verbosity by printing the message. Zero indicates
+        nothing is printed.
+
+    Attributes
+    ----------
+    regions_: Niimg-like object
+        list of separated regions with each region lying on a 3D volume
+        concatenated into a 4D Nifti like object.
+
+    index_: numpy array
+        array of list of indices where each index value is assigned to
+        each separate region of its corresponding family of brain maps.
+
+    signals_: numpy array
```

Comment:
Is this still current? I have the impression that we are no longer defining such an attribute (and I think that it is a good thing).